### PR TITLE
Amnestics

### PR DESCRIPTION
**Pull request by Affurit**
Amnestics, as per request of Lion. I'm fully aware that some of it is a bit snowflakey and could be cut down, but realistically, it doesn't NEED to be cut down.

### CODE REVIEW

**Comment by JakeDaBoss** [comment]
Wrong branch.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -1940,4 +1940,28 @@
...
 	result = /datum/reagent/nanoblood
 	required_reagents = list(/datum/reagent/dexalinp = 1, /datum/reagent/iron = 1, /datum/reagent/blood = 1)
-	result_amount = 3
\ No newline at end of file
+	result_amount = 3
+	
+/datum/chemical_reaction/amnestics/a
+	name = "Amnestics-A"
+	result = /datum/reagent/amnestics/a
+	required_reagents = list(/datum/reagent/ethanol = 20, /datum/reagent/water = 10, /datum/reagent/sugar = 10)
```
Comment (LYNXEMS) [inline code/modules/reagents/Chemistry-Recipes.dm]: Doesn't this look way too simple and not too realistic? I suggest we use materials unavailable in every household or present times, such as phoron in the recipes!

**Comment by Affurit** [comment]
NO, I CLICKED EXPERIMENTAL. WHAT THE FUCK

**Comment by JakeDaBoss** [comment]
WELL IT DIDN'T!

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 1 above)
Comment (JakeDaBoss) [reply code/modules/reagents/Chemistry-Recipes.dm]: That would need a massive amount of phoron every round and unnecessary tbh. I'm open to discussion though.

**Comment by Affurit** [comment]
I only did as Lion asked me. I don't like Phoron in recipes personally and think we should cut it out where we can, and where we can't, we should just remove the products themselves as they are even more unrealistic.

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 1 above)
Comment (LYNXEMS) [reply code/modules/reagents/Chemistry-Recipes.dm]: I don't say exactly phoron, just sugar, alcohol and water should not create basically anomalous reagents imho

Comment (JakeDaBoss) [reply code/modules/reagents/Chemistry-Recipes.dm]: The recipe might change over time, but for now, this is what it is. I think we've got a good base here. Unless @Affurit wishes to suggest changes?

**Comment by LYNXEMS** [comment]
I see some problems

We are breaking the canon severely in this case

Each class has different application method
A- Aerosol
B - Pills / Injection
C - Injection
D - 4 Injections
E - More Injections
F - Also Injection, experimental
And there is O

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm [hunk 2]
```diff
@@ -365,3 +365,59 @@
...
+	
+
+/datum/reagent/amnestics/a
+	name = "Class-A Amnestics"
+	taste_description = "something you forgot about already"
+	
+	
+/datum/reagent/amnestics/a/on_mob_life(mob/living/M)
+	to_chat(M, "<span class='notice'>You feel your memory drifting away...")
+	to_chat(M, "<span class='boldannounce'>You have lost all memory up until the point before the last experiment (if any) you were involved in began. You must roleplay accordingly.</span>")
```
Comment (steamp0rt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm]: ```You have lost all memory past the last experiment ```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm [hunk 3]
```diff
@@ -365,3 +365,59 @@
...
+	holder.remove_reagent(/datum/reagent/amnestics/a, volume)
+	
+/datum/reagent/amnestics/b
+	name = "Class-B Amnestics"
+	color = "#00D9D9"
+	taste_description = "something you forgot about already"
+	
+/datum/reagent/amnestics/b/on_mob_life(mob/living/M)
+	to_chat(M, "<span class='notice'>Your brain feels slightly emptier...")
+	to_chat(M, "<span class='boldannounce'>You have lost all memory up until the point when the round began and you woke up. You must roleplay accordingly.</span>")
```
Comment (steamp0rt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm]: ```You have lost all memory past the point when the round began and you woke up```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm [hunk 4]
```diff
@@ -365,3 +365,59 @@
...
+	holder.remove_reagent(/datum/reagent/amnestics/b, volume)
+
+/datum/reagent/amnestics/c
+	name = "Class-C Amnestics"
+	color = "#cd7f32"
+	taste_description = "something you forgot about already"
+	
+/datum/reagent/amnestics/c/on_mob_life(mob/living/M)
+	to_chat(M, "<span class='notice'>Memories are ripped out of your head!")
+	to_chat(M, "<span class='boldannounce'>You have lost all memory up until the point when you arrived at the foundation, and have no idea how you got here. You must roleplay accordingly.</span>")
```
Comment (steamp0rt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm]: ```You have lost all memory past the point when you arrived at the foundation, and have no idea how you got here```

**Pull request merged**